onadata/apps/api/tests/viewsets/test_abstract_viewset.py

```diff
@@ -370,6 +371,20 @@ def _publish_xls_form_to_project(
                 # pylint: disable=attribute-defined-outside-init
                 self.form_data = response.data
 
+        data_value = "editor|dataentry|readonly"
+        metadata = self.xform.metadata_set.get(data_type="xform_meta_perms")
+        serializer = MetaDataSerializer(
+            metadata,
+            data={
+                "data_value": data_value,
+                "data_type": "xform_meta_perms",
+                "xform": self.xform.id,
+            },
+        )
+
+        if serializer.is_valid():
```

Comment:
We can omit serializer.is_valid() since for tests it doesn't matter if an exception is thrown

Reply:
@FrankApiyo Ok noted